src/Api/Serializer/BasicDiscussionSerializer.php

```diff
@@ -36,6 +36,7 @@ protected function getDefaultAttributes($discussion)
         return [
             'title' => $discussion->title,
             'slug'  => $discussion->slug,
+            'excerpt' => $discussion->excerpt(),
```

Comment:
What I hadn't considered when [suggesting to encapsulate this in the model](https://github.com/flarum/core/pull/1948#issuecomment-578299506) is that we now expose this in the API, even though it's sort of an implementation detail. And I strongly dislike adding things to APIs unless we have a direct use-case for them - you are bound to support them forever.

Thoughts?

Reply:
While I generally agree with the sentiment, and also think that bloating core's API can be problematic, I also have a hard time imagining a situation where discussions cease to have text and this is no longer relevant.

I don't suppose there's any other way we could get the description instance itself into Forum\Content\Document?

Reply:
Just thought of something while trying to figure out #2177: The Tag Content pulls in an instance of the tag: https://github.com/askvortsov1/tags/blob/285622d51999d012e368d9d82a0e8f5bbe4777ef/src/Content/Tag.php#L44-L71

We could do the same here, then we don't need add anything to the API at all.